### ISSUE TITLE
[Bug][FlexibleColumnLayout][RC.11] accessibilityTexts does not work in FlexibleColumnLayout Component

### ISSUE THREAD

**Issue by Hubery-Shen** [SAP/ui5-webcomponents#2631]
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.

**Bug Description**
**Version: 1.0.0-rc.11**
accessibilityTexts does not work in FlexibleColumnLayout Component

**My code:**

```
<ui5-flexible-column-layout id="fcl" style={{height: "600px"}}
  accessibilityTexts={{
      startArrowLeftText: 'startArrowLeftText1',
      startArrowRightText: 'startArrowRightText1',
      endArrowLeftText: 'endArrowLeftText1',
      endArrowRightText: 'endArrowRightText1'
  }}></ui5-flexible-column-layout>
```

I set the accessibilityTexts attribute to FlexibleColumnLayout component but it does not work.

![Screen Shot 2021-01-06 at 2 20 15 PM](https://user-images.githubusercontent.com/63704163/103737905-18386980-502e-11eb-9f70-3951652358b8.png)

**I have wrote an example in this repo:** https://github.com/Hubery-Shen/UI5-WebComponent-Examples.git

**Comment by ilhan007** [comment]
Hello @Hubery-Shen it seems the JSX tries to set the accessibilityTexts as attribute. Could you try to manually set the property after the view mounts by:

```js
    fclRef.accessibilityTexts = {} // your object
```

**Comment by Hubery-Shen** [comment]
Hi @ilhan007 
OK, I will try it

Thanks

**Comment by ilhan007** [comment]
Hi @Hubery-Shen 
Is it working now?

**Comment by Hubery-Shen** [comment]
Hi @ilhan007 

Yes, it works.

Thanks you very much. I will close this issue
